ada_2_fhir-r4/zibs2020/ada_project_drivers/MedicationContraIndication-driver.xsl

```diff
@@ -25,6 +25,7 @@ The full text of the license is available at http://www.gnu.org/copyleft/lesser.
     xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
     version="2.0">
     
+    <!--    <xsl:import href="../payload/zibs2020_2_fhir.xsl"/>-->

```

Comment:
Ik weet niet of zibs2020_2_fhir de juiste plek is, maar ik wilde ergens een stylesheet waarin ik dingen kan toevoegen specifiek voor dit project.